source/developer/interactive-dialogs.rst

```diff
@@ -274,6 +274,14 @@ Moreover, Mattermost also allows the integration itself to perform input validat
 
   {"errors": {"num_between_0_and_10": "Enter a number between 0 and 10."}}
 
+The integration may also return a generic error message to the user that is not attached to a specific field. This can be done by responding to the dialog submission request with a JSON body containing an `error` field. The `error` field should contain a string with the error message to display to the user. For example, if a server-side error occurs, you can return a message explaining it:
+
+.. code-block:: json
+
+  {"error": "Failed to fetch additional data. Please try again."}
+
+Support for generic error messages was added in Mattermost 5.16.
```

Comment:
@pbitty This feature won't make it into 5.16. Would you please change this to 5.18?
```suggestion
Support for generic error messages was added in Mattermost 5.18.
```